server/telemetry.py

```diff
@@ -0,0 +1,51 @@
+from collections import deque
+from datetime import datetime
+from flask_socketio import SocketIO
+
+class TelemetryController:
+    def __init__(self, socketio: SocketIO) -> None:
+        self.incoming_data = deque([], maxlen=5)
```

Comment:
Potential rename of self.incoming_data to self.telemetry_data or self.current_telemetry_data or another better name, since telemetry may be used by other functions for routing and fail safe checks, and not just to record incoming data

Reply:
Updated